src/main/java/fr/polytech/sim/log/NamedLogger.java

```diff
@@ -16,4 +16,12 @@ protected NamedLogger(String name) {
         this.name = name;
     }
 
+    public void log(String format, Object... args) {
+        String entry = String.format(format, args);
+        String message = String.format("%s\t%s\n", this.name, entry);
+        this.write(message);
+    }
+
+    public abstract void write(String message);
```

Comment:
Cette méthode devrai avoir un accès `protected`.
```suggestion
    protected abstract void write(String message);
```